tracer/src/Datadog.AutoInstrumentation.NativeLoader/cor_profiler.cpp

```diff
@@ -782,7 +844,17 @@ namespace datadog::shared::nativeloader
     HRESULT STDMETHODCALLTYPE CorProfiler::GetReJITParameters(ModuleID moduleId, mdMethodDef methodId,
                                                               ICorProfilerFunctionControl* pFunctionControl)
     {
-        RunInAllProfilers(GetReJITParameters(moduleId, methodId, pFunctionControl));
+        if (isWriteInstrumentationToDiskEnabled)
+        {
+            instrumented_assembly_generator::InstrumentedAssemblyGeneratorCorProfilerFunctionControl* instrumentationVerificationFunctionControl =
```

Comment:
This feels like a leak:
this pointer is passed to the profiler and tracer. Only the tracer will keep the pointer, but never delete it.
What is the life duration of this pointer ?

Reply:
Please see the change in ebde2f57c8ef03fd167862f981abd5c4958c222a I think it should be ok now.